.github/workflows/build-package-norelease.yml

```
@@ -43,6 +43,9 @@ jobs:
 
       - name: build for osx
         run: dotnet publish -c Release --runtime osx-x64 -o build streamdeck-vtubestudio
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)